src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php

```diff
@@ -84,6 +84,16 @@ public function getContentPath($route, $webspaceKey = null, $locale = null, $dom
             $domain,
             $scheme
         );
+        if ($allDomains && !$url) {
```

Comment:
I don't really understand what the `$allDomains` is doing and what it is needed for... It sounds to me like it should return an array, because all could also mean multiple. But from what I am seeing it is still returning only one URL.